strapi/strapi
# Add confirm alert on close modal (upload) & bump DS version

Signed-off-by: HichamELBSI <elabbassih@gmail.com>



### What does it do?

- Add confirm alert on close modal (Edit asset form & before file upload)
- Add tests
- Bump DS version

### How to test it?

- Change a value in "Edit asset modal" form
- Try to close before submit
- A confirm alert should be visible
